tests/Kernel/CallForProposalsRenderTest.php

```diff
@@ -0,0 +1,280 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_theme\Kernel;
+
+use Drupal\Core\Datetime\DrupalDateTime;
+use Drupal\datetime\Plugin\Field\FieldType\DateTimeItemInterface;
+use Drupal\node\Entity\Node;
+use Drupal\Tests\oe_theme\PatternAssertions\ListItemAssert;
+use Drupal\Tests\oe_theme\PatternAssertions\FieldListAssert;
+use Drupal\Tests\oe_theme\PatternAssertions\PatternAssertState;
+use Drupal\Tests\user\Traits\UserCreationTrait;
+use Drupal\user\Entity\User;
+use Symfony\Component\DomCrawler\Crawler;
+
+/**
+ * Tests call for proposals rendering.
+ */
+class CallForProposalsRenderTest extends ContentRenderTestBase {
+
+  use UserCreationTrait;
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'options',
+    'field_group',
+    'composite_reference',
+    'oe_time_caching',
+    'oe_content_reference_code_field',
+    'oe_content_departments_field',
+    'oe_content_call_proposals',
+    'oe_theme_content_call_proposals',
+    'datetime_testing',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp(): void {
+    parent::setUp();
+
+    $this->installConfig([
+      'composite_reference',
+      'oe_content_reference_code_field',
+      'oe_content_departments_field',
+      'oe_content_call_proposals',
+      'oe_theme_content_call_proposals',
+    ]);
+
+    module_load_include('install', 'oe_content');
+    oe_content_install();
+
+    // Set current user to UID 1, so that by default we can access everything.
+    $account = User::load(1);
```

Comment:
Instead of doing this maybe it would be better to use setUpCurrentUser([], [], TRUE) so that both the admin and anonymous users are generated (just to avoid possible errors),